### ISSUE TITLE
SisLovesMe Matching

### ISSUE THREAD

**Issue by claygoldfinch** [PAhelper/PhoenixAdult.bundle#154]
Pulling data from "https://www.sislovesme.com/SCENEID/banner/1" rather than "https://www.sislovesme.com/t2/?single=SCENEID" may resolve some matching issues. 

For example:
https://www.sislovesme.com/7/banner/1
vs
https://www.sislovesme.com/t2/?single=7

**Comment by PAhelper** [comment]
I'll have to check and see if there are any examples where the current one is superior to your new suggestion. Don't want to trade one problem for another. Might be able to use both with a bit of code... If one fails, use the other.

**Comment by kamuk90** [comment]
update posters:
28-29 
```
   Log("urlName: " + detailsPageElements.xpath('//video[@id="preview"]')[0].get("poster").split('/')[5])
   urlName = detailsPageElements.xpath('//video[@id="preview"]')[0].get("poster").split('/')[5]
```

to

``` 
   Log("urlName: " + detailsPageElements.xpath('//video[@id="main-movie-player"]')[0].get("poster").split('/')[5])
    urlName = detailsPageElements.xpath('//video[@id="main-movie-player"]')[0].get("poster").split('/')[5]
```

55 line:
```       
art.append("http:" + detailsPageElements.xpath('//video[@id="preview"]')[0].get("poster"))
```

to

```
art.append("http:" + detailsPageElements.xpath('//video[@id="main-movie-player"]')[0].get("poster"))
```

**Comment by claygoldfinch** [comment]
Resolved both of these issues in PR #186. Thanks for the heads up, @kamuk90!

**Comment by VictorLugoSr** [comment]
https://rapidgator.net/file/dea343c2dd7012be429b878a1082402d/Vina_Sky_-_Valenteen_Devil.mp4.html
https://rapidgator.net/file/dd7d02d4af014ca21bd4b65bcc2120c3/Vina_Sky_-_Young_&_Glamorous_10.mp4.html
https://rapidgator.net/file/e0e5cf487f68a8a3e2ee4c2a5905542e/Vina_Sky_-_Circling_The_Rim.mp4.html
https://rapidgator.net/file/db4983324239f815d1392d69c4724399/Vina_Sky_-_Vina's_Special_Day.mp4.html
https://rapidgator.net/file/ad9d68126074ac14a4142125e54df03b/Vina_Sky_-_Repaying_The_Babysitter.mp4.html
https://rapidgator.net/file/828504aa7b62378852987eb5037636de/Megan_Rain_-_Teens_Love_Huge_Cocks.mp4.html
https://rapidgator.net/file/d163ea828f3878d19e6e7814e040c4dd/Vina_Sky_-_Stranded_Sky.mp4.html
https://rapidgator.net/file/1d29915ff6490e306996af0ef2b90d8b/Vina_Sky_-_Cant_Get_Enough.mp4.html
https://rapidgator.net/file/26570068c9be763f04d0bb2944211fd4/Meet_Elsa.mp4.html
https://rapidgator.net/file/569e082902af5df1b15ba16f1bb0454e/Vina_Sky_-_A_Nerd_In_Need_Gets_The_D.mp4.html
https://rapidgator.net/file/199261d7e7e371dbd59f8ddd4627d839/Vanna_Bardot_-_Fuck_Me_Twice.mp4.html
https://rapidgator.net/file/06b9343a760086e8e2e62e44cb82ea20/Vanna_Bardot_-_Casual_in_Love.mp4.html
https://rapidgator.net/file/bc97dc5f7f6535364c5685b7dac25ec4/Vanna_Bardot_-_Naughty_Nurse.mp4.html
https://rapidgator.net/file/d28c47bdbebe6d7d89350195f8ebccc7/Sydney_Cole_-_Daddys_Little_Princess.mp4.html
https://rapidgator.net/file/69e37814892b647c67b4ebc8b8013050/Vanna_Bardot_-_Eat_My_Cookie.mp4.html
https://rapidgator.net/file/f9d3a86fb0e515bd285b6f4abfa2a35e/Naomi_Woods_-_Property_Sex.mp4.html
https://rapidgator.net/file/ec30eb0d6f0cfc8da39d88a26e2292e2/Natalia_Queen_-_DontBreakMe.mp4.html
https://rapidgator.net/file/af7879ee98bedcd311b668563274ffe4/Lana_Rhoades_-_CuckoldSessions.mp4.html
https://rapidgator.net/file/bc119891bc877a05212cc5a118340c13/Hannah_Hays_-_Teens_Love_Huge_Cocks.mp4.html
https://rapidgator.net/file/6315ab12386e54f459d81cc3c4576739/Naomi_Woods_-_DontBreakMe.mp4.html
https://rapidgator.net/file/0d7a1497a3bd6d17234e4d82ac85baca/Elsa_Jean_-_DontBreakMe.mp4.html
https://rapidgator.net/file/1b10ccdfa42d3eb6193213a361ea002f/Elsa_Jean_-_Big_Game_Tiny_Hole.mp4.html
https://rapidgator.net/file/a4379b9eab7c23ffe0a8b60ecaad6e38/Maria_Kazi_-_DadCrush.mp4.html
https://rapidgator.net/file/e3eac956ce224e2a2a81f235f1455528/Leana_Lovings_-_Teens_Love_Huge_Cocks.mp4.html
https://rapidgator.net/file/f727c51df25c0ad2d242b3bdfcb9c300/Lana_Rhoades_-_SisLovesMe.mp4.html
https://rapidgator.net/file/3e85d5bb7f76fd4320757ab42b4e00f4/Kylie_Rocket_-_Teens_Love_Huge_Cocks.mp4.html
https://rapidgator.net/file/ff5f3c6f7a454504014879d47035ef33/Lana_Rhoades_-_Agrees_to_Fuck_Her_Brother.mp4.html
https://rapidgator.net/file/fd809e01df8bd5ee7e6e6d0777b8dc90/Emily_Willis_-_That_70s_Ho_Hyde_Moves_On_In.mp4.html
https://rapidgator.net/file/aa7a175ba594c507c05908f2f366b07e/Kali_Roses_-_DadCrush.mp4.html
https://rapidgator.net/file/a8c51d0abfbb74df197f5f06c1d6e8d5/Emily_Willis_-_Holding_Out_&_Giving_In.mp4.html
https://rapidgator.net/file/5fca6055827ef451120e238cd25beb7e/Gabbie_Carter_-_Teens_Love_Huge_Cocks.mp4.html
https://rapidgator.net/file/1316d44d4fd46f1b94a17599ed65e9b9/Gabbie_Carter_-_My_Sisters_Hot_Friend.mp4.html
https://rapidgator.net/file/e6f5ffeed99cf093455be1fcdeb765f8/Emily_Willis_-_Sneaky_Stepfather.mp4.html
https://rapidgator.net/file/65af097d5d90db128030eee28fe0fa10/Emily_Willis_-_Dangerous_Women_Part_2.mp4.html
https://rapidgator.net/file/c2c328326caa153bfcc19c4bb2d44767/Emily_Willis_-_Creeping_Stepsis.mp4.html
https://rapidgator.net/file/074e9132ed0fa3fd2a5af23612d61b16/Emily_Willis_-_Crack_That_Whip.mp4.html
https://rapidgator.net/file/0172cd084806f6b3b9b092e246fb8d5c/Bailey_Brooke_-_DadCrush.mp4.html
https://rapidgator.net/file/e6c85ce27477b3511fe524fab0c03381/Blair_Williams_-_Only_Teen_Blowjobs.mp4.html
https://rapidgator.net/file/f1d36fc01fbbf5238de1fcac9aaa9928/Dillion_Harper_-_Secret_Seduction.mp4.html
https://rapidgator.net/file/2d89b029a2ab06792ffead542f903fc0/Dillion_Harper_-_No_Time_Like_The_Present.mp4.html
https://rapidgator.net/file/a6718a41517468f1df7c523879e24f98/Dillion_Harper_-_Ruxrose.mp4.html
https://rapidgator.net/file/194b797657a76a50a167bacd3646921a/Bailey_Brooke_-_Family_Therapy.mp4.html
https://rapidgator.net/file/a4793ff1403e5eefcf7074eb9eb48572/Blair_Williams_-_SisLovesMe.mp4.html
https://rapidgator.net/file/aa24513519458a97f9c726d9520a38da/Dillion_Harper_-_Eves_Secret_2.mp4.html
https://rapidgator.net/file/62f8f8ff58a4282b1aeab627bb3b3fa1/Dillion_Harper_-_Camping.mp4.html
https://rapidgator.net/file/43399e4a70e65ba836dcce788c968d7c/DadCrush.6.mp4.html
https://rapidgator.net/file/a5200150e8eaa0376037c117355ad09e/DadCrush.4.mp4.html
https://rapidgator.net/file/9f1a9f4743b585f02882bc81e2c9f99d/DadCrush.2.mp4.html
https://rapidgator.net/file/ef038666c946fcc96cc4dbaedddb2af0/DadCrush.1.mp4.html
https://rapidgator.net/file/01f65e20b192b31f6bc0e9b0ad329754/Blair_Williams_-_Household_Piping.mp4.html
https://rapidgator.net/file/585ba4405b6d071d3e126ecf6833ca19/Nicole_Aniston_-_PervMom.mp4.html
https://rapidgator.net/file/1e25966c28483eebd85dd0905d53d168/Mazy_Myers_-_Spank_Bang.mp4.html
https://rapidgator.net/file/c9622568c9777076773258bc4a6f62ae/Victoria_Steffanie_-_Stepsister.mp4.html
https://rapidgator.net/file/b580976d73771125f1b69d2eb8948c9a/Taboo_Heat_Family_Obligations.mp4.html
https://rapidgator.net/file/231f08d34e6755212dac222e76ef619c/Sydney_Cole_-_My_Sisters_Cold_Feet.mp4.html
https://rapidgator.net/file/47d25c046eb430ba30cc591bc238e9ac/Tiffany_Tatum_-_SneakySex.mp4.html
https://rapidgator.net/file/2b5493640e4215e7d61a795f461a27fe/Sydney_Cole_-_Stepbrother_Amour.mp4.html
https://rapidgator.net/file/52246aa4de0791777efff1e9598f1533/Stepsis_Is_Addicted_To_My_Dick.mp4.html
https://rapidgator.net/file/8832abe40a7a118214fcc05ab6ad15b8/Susanne_Brend_-_White-Hot_Nurses_8.mp4.html
https://rapidgator.net/file/a8af9b88c371db5a2ca4eac7f164042d/Stacy_Cruz_-_Teach_Me_How_To_Play.mp4.html
https://rapidgator.net/file/aca352db37276f4f50c03b2d45f7dd08/Sandra_Luberc_-_Tiny_Russian_Pov.mp4.html
https://rapidgator.net/file/0ab1b710ba81349cb5fb66aca518a50b/SpyFam_Dad_Busy_Watching_Game.mp4.html
https://rapidgator.net/file/d299d4f87e9dbfe0f48e06affb4346c8/Stacey_Saran_&_Danny_D_-_A_Very_Neighborly_Affair.mp4.html
https://rapidgator.net/file/0d502bf17cae1771c82643f1b6fa2b1e/Sisters_Lucky_Day.mp4.html
https://rapidgator.net/file/04ee9c4294dc10c26885ce808906be61/Sis,_You_Are_Way_Better_Than_My_GF!!.mp4.html
https://rapidgator.net/file/27066970584fe90cf419cf7e45dbdd98/PervMom_Cory_Chase.mp4.html
https://rapidgator.net/file/772eda2b46f655f7036b795668839f72/She_loves_dick_in_her_mouth.mp4.html
https://rapidgator.net/file/6f9aa307ea939ee6cf5065a771353e4b/PureTaboo_-_Being_Neighborly.mp4.html
https://rapidgator.net/file/333f9e0f07d9b9b9fe6e731b4dd006c7/Eva_Lovia_-_Teens_love_huge_cocks.mp4.html
https://rapidgator.net/file/04387d56236f77f6168107f0de5f5ec5/Eva_Lovia_-_Her_Fantasy_Ball.mp4.html
https://rapidgator.net/file/14fce0c2de621f05fded5edae752cb26/Penny_Barber_-_Full_Med.mp4.html
https://rapidgator.net/file/84c965617a292df4d0004ab6f69da044/Open_Family_-_Natasha_Nice.mp4.html
https://rapidgator.net/file/04d0695eba1c06745dbea67183a184a2/Nicole_Doshi.mp4.html
https://rapidgator.net/file/b7aec13471ea5456dfd459c1385b4e93/My_Stepsis_Pajama_Panties.mp4.html
https://rapidgator.net/file/e6e5625b0c7ec0059efc77f415d8f363/Janice_Griffith_-_Too_Big_For_Teens.mp4.html
https://rapidgator.net/file/9f12c4cf1652447d2e3d6ebd1384320e/MomsTeachSex_-_Let_Mommy_Help_You.mp4.html
https://rapidgator.net/file/2a097fff6c478537b5e2115927aecae7/Mommy_Got_Boobs.mp4.html
https://rapidgator.net/file/80fa509d14e5050e0f24f91164ba8a47/Michelle_Anderson_-_Full_Low.mp4.html
https://rapidgator.net/file/714adb6ee50517106b8ac710984fd289/Mercedes_-_Sweet_Cheeks.mp4.html
https://rapidgator.net/file/e7adcbb220977a3178e2fa45a6d0fa6f/Layla_Ray.mp4.html
https://rapidgator.net/file/2d645d5121387b11bdf345d50dc766f9/Jayna_Woods_-_Sweet_Cheeks.mp4.html
https://rapidgator.net/file/064aae5d1acabe9390a89d571d18fd62/Jada_Stevens_-_Blackened.mp4.html
https://rapidgator.net/file/999d1c936be374a7dabc21f96a926075/Jasmine_Grey_-_Valentines_Day-Surprise.mp4.html
https://rapidgator.net/file/38a7bfb89cf6bf47ece9bc3dd3b0c50c/Haley_Reed_-_Dark_Chocolate_Easter_Anal.mp4.html
https://rapidgator.net/file/572dfcb7c0bc1f78c66d288460f46271/Fix_Me_A_Snack.mp4.html
https://rapidgator.net/file/30c1e2e69a6b7ce127bb1bcb9567630d/Gianna_Dior_-_Get_Me_Wet.mp4.html
https://rapidgator.net/file/704cbe0bf660244ffb7560eb65cb5a0f/Had_Some_Fun_Gotta_Run.mp4.html
https://rapidgator.net/file/cf626487fa48450a2876b7ea4fc98bc0/Ariana_Marie_-_Step-Sister_Waits_For_Parents_To_Leave.mp4.html
https://rapidgator.net/file/c9481ed96d9c65e3724a71454c1626be/FamilyStrokes_-_Faking_Out_Your_Father.mp4.html
https://rapidgator.net/file/cda44dc4e1c3ccc6201b2823f2ca3e1e/Family_Therapy_-_A_Son's_Progress.mp4.html
https://rapidgator.net/file/18e8f6d8df690f7d7df23486e4a3bd25/Eliza_Ibarra_-_No_Interruptions.mp4.html
https://rapidgator.net/file/028036a7b7d4040b24f3ab57ccbb0778/Eva_Elfie_-_Cumming_In_Ecstasy.mp4.html
https://rapidgator.net/file/9d7875adf0b16a050cb2ee2e328d7132/Easter_Bunny_(Creepy_Uncle)_Banged_Her.mp4.html
https://rapidgator.net/file/a007b18a908fa110702241c00937b053/Black_Ambush_-_Ivy.mp4.html
https://rapidgator.net/file/aa3f7f0c205037f657e808e7d3d6f4c7/Eliza_Ibarra_-_Graphic_Content_-_Scene_4.mp4.html
https://rapidgator.net/file/716c13bd01193dbedb259aa929e033bc/Cory_Chase_-_Taboo_Heat.mp4.html
https://rapidgator.net/file/610e22dfda9e5b791bbb925c14a0b5a3/Clara_Trinity_-_Winter_Princess.mp4.html
https://rapidgator.net/file/05d0c462e249e82165c2027ac9b613d3/College_Teen_Gets_a_Load_of_Cum_in_her_Mouth.mp4.html
https://rapidgator.net/file/464c6456f4228ba1b180ad4ab43ea661/Aria_Sky_-_My_Sisters_Hot_Friend.mp4.html
https://rapidgator.net/file/c373a50bbe9bb5918b9e77616caf51a9/BrattySis.Jealous.Step.Brother.mp4.html
https://rapidgator.net/file/62dcd48f91cfffe1e87b6ed39f4be2b0/BrattySis_-_My_Sister_And_Her_Friend.mp4.html
https://rapidgator.net/file/1d1ade7208f44d4e5d833e3e7d98707b/Blake_Blossom_-_Ping_Pong_Cum.mp4.html
https://rapidgator.net/file/17448efd26c6e2391e536f0800a6aa87/BBCPIE.mp4.html
